### PR TITLE
New Visualization Navigation

### PR DESCRIPTION
**Pull request by achang28**
As usual, not yet ready to merge.

### CODE REVIEW

**Comment by achang28** [comment]
@jingchunzhu @maryjgoldman pull from my repo and checkout the KM Plot button (https://github.com/achang28/ucsc-xena-client/tree/NewNavigation).  Still a big Work in Progress, but at the very minimum the KM Plot graph is showing up...

**Comment by jingchunzhu** [comment]
I clicked on KM Plot, but no KM plot showing up. This is on TCGA Breast cancer cohort.

**Comment by jingchunzhu** [comment]
The new black navbar really brothers me, I like our current production's "non-inverse" (or default)  look and feel.   Attached a diff to use what's on production look.
[nav.js.diff.txt](https://github.com/ucscXena/ucsc-xena-client/files/205903/nav.js.diff.txt)

The diff requires a js/nav.css file, contents below

.navbar .nav li a:hover
{
    background-color:#e7e7e7
}

.navbar-default .navbar-nav > .active > a, .navbar-default .navbar-nav > .active > a:hover, .navbar-default .navbar-nav > .active > a:focus
{
    color:#ff6600;
    background-color:white
}

File: js/ChartView.js [hunk 1]
```diff
@@ -2,9 +2,12 @@
...
 var React = require('react');
+var _ = require('./underscore_ext');
 
 var ChartView = React.createClass({
-	shouldComponentUpdate: function () {
+	shouldComponentUpdate: function (newProps) {
+		if (!_.isEqual(newProps.appState, this.props.appState))
+			debugger;
 		return false;
 	},
```
Comment (achang28) [inline js/ChartView.js]: @acthp Do you know exactly what the application is considering when determining whether to render the actual chart, or the appropriate error message?

From the img below, while on the Chart view, switching cohort does not dismiss the previous chart view.  I went from 'TCGA PANCAN12' to 'ccle'.  The master branch currently behaves this same way as well (I turned off the disable parameter on the Cohort drop-down menu in order to test this issue out).  From the above code changes, I was trying to see what differences to consider between new appState vs. old appState.

incorrect render

**Comment by achang28** [comment]
@jingchunzhu Pull the latest codebase.  I incorporated the latest updates from the Master branch in production, and also incorporated the subtle visual changes you mentioned 2 days ago.

**Comment by jingchunzhu** [comment]
good work-in-progress.

Jing

File: js/controllers/common.js [hunk 2]
```diff
@@ -63,6 +63,7 @@ var setCohort = (state, cohort) =>
 				"columnOrder", [],
 				"data", {},
 				"datasets", null,
+				"mode", 'heatmap',
```
Comment (achang28) [inline js/controllers/common.js]: @acthp Rather than hard-coding the value, where would I source the default ```mode``` from?

**Pull request closed without merging**